source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,56 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes of data in MongoDB, such as changes to a
+collection, database, or deployment by opening a :manual:`Change
+Stream</changeStreams/>`. A ``Change Stream`` allows applications to
+watch for changes to data and react to those changes. You can open a
+``Change Stream`` by calling the :node-api:`watch()
+</Collection.html#watch>` method on a collection, database, or
+deployment. 
+
+The ``watch()`` method takes an optional pipeline as the first parameter
+that is an array of stages to filter the :manual:`change events
+</reference/change-events/>` output. Also, ``watch()`` can take in
```

Comment:
You might consider "can take an additional options object" or "can accept an additional options object". I think "take in" has other meanings which makes it ambiguous.

Reply:
altered to:
"Also, ``watch()`` can take an additional options object as the second parameter."